specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2025-03-02-preview/managedClusters.json

```diff
@@ -2241,6 +2241,21 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/": {
+      "post": {
+        "tags": [
+          "AdvancedNetworkingTransitEncryption"
+        ],
+        "operationId": "AdvancedNetworkingTransitEncryption_WireGuard",
+        "summary": "Enable WireGuard for Advanced Networking Transit Encryption",
+        "description": "Enables WireGuard for Advanced Networking Transit Encryption. This API is only available for clusters with Advanced Networking enabled. Refer to https://docs.cilium.io/en/latest/security/network/encryption-wireguard/ on use cases and implementation details",
+        "x-ms-examples": {
+          "Stop Managed Cluster": {
+            "$ref": "./examples/AdvancedNetworkingTransitEncryption.json"
+          }
+        }
+      }
+    },
```

Comment:
you are not adding a new route, please add your example to the PUT method of  
`/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}`